src/App.js

```diff
@@ -1,96 +1,106 @@
     setIsRoomEnlighten(true); //this is NOT a toggle, changed to false only in changePosition above
   }
 
-  function processingLatestSpell(input) {
-    const treatedInput = input.trim().split(' ');
-    const displayedInput = treatedInput.filter(inputUnit => inputUnit.length > 0); //this is used later aswell
-    const preparedInput = displayedInput.map(checkedInputPart => checkedInputPart.toLowerCase());
+  function processingLatestSpell(instruction) {
+    const treatedInstruction = instruction.trim().split(' ');
+    const instructionValues = treatedInstruction.filter(inputUnit => inputUnit.length > 0); //this is used later aswell
+    const preparedInstruction = instructionValues.map(checkedInputPart => checkedInputPart.toLowerCase());
 
-    const commandMessage = findCommandMessage(preparedInput[0]);
+    const commandMessage = findCommandMessage(preparedInstruction[0]);
     const commandHasError = commandMessage.includes('ERROR');
 
-    const spellEffects = executeSpell(preparedInput, currentPosition, changePosition, changeEnlighten);
+    const spellEffects = executeSpell(preparedInstruction, currentPosition, changePosition, changeEnlighten);
     const {spellEffectMessage, spellEffectOutput, spellEffectHasError} = spellEffects;
 
     setSavedOrders([
       {
         id: nanoid(),
-        inputValues: displayedInput,
-        commandMessage: commandMessage,
-        commandHasError: commandHasError,
-        spellEffectMessage: spellEffectMessage,
-        spellEffectOutput: spellEffectOutput,
-        spellEffectHasError: spellEffectHasError,
+        instructionValues,
+        commandMessage,
+        commandHasError,
+        spellEffectMessage,
+        spellEffectOutput,
+        spellEffectHasError,
       },
       ...savedOrders,
     ]);
 
     const List = ref.current;
-    const topPos = List.offsetTop;
-    List.scrollTop = topPos;
+    const bottomPos = List.offsetBottom;
+    List.scrollTop = bottomPos;
   }
 
   return (
     <OrganizingMain>
       <LogoImg src={require('./images/logo1.png')} alt="" width="125" height="125" />
-      <h1>
-        <u>SHELL_WIZARD</u>
-      </h1>
+      <UnderlinedHeading> SHELL_WIZARD </UnderlinedHeading>
       With this App you can learn about your first steps with the zshell (what you want to learn, if you want to learn
       how to code!)
       <br />
       You are a Wizard searching for wisdom in the inner depths of a cursed maze. Your only chance to find your way
       through this, is to type in Spells (that resemble the zShell-commands) to the textfield in the area below.
-      <ol>
-        <li>
-          <CodeSpan>pwd</CodeSpan> - to check on your current position.
-        </li>
-        <li>
-          <CodeSpan>ls</CodeSpan> - to find new rooms. A second argument is optional.
-        </li>
-        <li>
-          <CodeSpan>cd</CodeSpan> - to move from one room to another. A second argument is required.
-        </li>
-      </ol>
+      <StyledDl>
```

Comment:
I'm happy to see a  Description List Element, it is very rare creature 🦄

Reply:
This was of course jerrys recommendation 😜